Tests/AsyncAlgorithmsTests/TestZipLatestFrom.swift

```diff
@@ -0,0 +1,782 @@
+//
+//  TestZipLatestFrom.swift
+//  
+//
+//  Created by Thibault Wittemberg on 01/04/2022.
+//
+
+@preconcurrency import AsyncAlgorithms
```

Comment:
oof wat?
XCTest I get, but the framework we are building for async/await stuff; im sure this is just a copy/paste error?

Reply:
removed :-) It was due to a lack of conformance to Sendable